packages/common/validators/src/validators/is-birth-date-6.spec.ts

```diff
@@ -11,6 +11,9 @@ describe('isBirthDate6', () => {
     expect(isBirthDate6('foobar')).toEqual(false);
     expect(isBirthDate6('000000')).toEqual(false);
     expect(isBirthDate6('960732')).toEqual(false);
+    expect(isBirthDate6('951301')).toEqual(false);
+    expect(isBirthDate6('950230')).toEqual(false);
+    expect(isBirthDate6('950431')).toEqual(false);
```

Comment:
I think there are too many expect phrases to verify on one test suite(`it`). 
Why don't you split it up in different test suite?

Reply:
@okinawaa I agree with you. [test: split test case](https://github.com/toss/slash/pull/420/commits/9af5499148cf0c6ec8d50327488d62a45b1e3505)